### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by mh4ck-Thales** [containers/podman-compose#1079] [General]
Hi, I'm wondering if a new release of podman-compose is planned soon? I'm interested in [this bug fix](https://github.com/containers/podman-compose/pull/978) among others

May be related to https://github.com/containers/podman-compose/discussions/702 as I wasn't able to find information about a release schedule anywhere

**Comment by ArthurHNL** [comment]
+1, I'm interested in #971.

**Comment by flixman** [comment]
@p12tic Sorry for pulling you here: do you know anything about when the next official release will be available?

**Reply by p12tic** [reply]
It's just a question of me spending half an hour on doing the release. 1.3.0 has been released on github, it will take a couple of days for it to appear on Pypi.
